src/GovUk.Education.ExploreEducationStatistics.Content.Services/PublicationService.cs

```diff
@@ -197,8 +198,10 @@ public async Task<Either<ActionResult, PaginatedListViewModel<PublicationSearchR
             _ => throw new ArgumentOutOfRangeException(nameof(sort), sort, message: null)
         };
 
-        // Then sort by publication id to provide a stable sort order
-        orderedQueryable = orderedQueryable.ThenBy(result => result.Value.Id);
+        // Order by release type, then publication ID to provide a stable sort order
+        orderedQueryable = orderedQueryable
+            .ThenByReleaseType()!
+            .ThenBy(result => result.Value.Id);
```

Comment:
I think there is little point ordering by release type if you're searching by `Relevance` or `Title`, since that would only make a difference if we had identical titles, which even if it happened would be a rare instance.

So with it being a small change to make, I'd be inclined to move this ordering into the switch statement case `Published`. 

This endpoint is doing a lot of stuff and anything we can do to improve the performance is worthwhile, I think. Although I can imagine if you actually measured the performance difference when searching by text you'd find it makes little to no difference.